server/README.md

```diff
@@ -0,0 +1,16 @@
+# The Custom Model Aggregator
+
+## About
+This folder contains all the necessary files to build and run a simple model aggregator in an enclave environment. The aggregator takes the element-wise sum of the variables received from multiple clients, weighs them by local iterations, then adds them to the existing model. All sensitive computation is done in an enclave environment, with encryption/decryption being performed inside.
+
+## Building
```

Comment:
Just running `make` will accomplish steps 1 and 4 as well, right?

Reply:
It will accomplish 3-4

Reply:
I don't think so.. the enclave/CMakeLists.txt file should run the oeedger8r automatically, and it will sign the enclave binary as well. So steps 1 and 5 below should be redundant.